modules/checklist/assets/js/app/components/success-message.js

```diff
@@ -0,0 +1,42 @@
+import { Button, Card, CardActions, CardContent, CardMedia, Typography } from '@elementor/ui';
+
+const SuccessMessage = () => {
+	const hideChecklist = () => {
+		$e.run( 'document/elements/settings', {
+			container: elementor.settings.editorPreferences.getEditedView().getContainer(),
+			settings: {
+				show_launchpad_checklist: '',
+			},
+			options: {
+				external: true,
+			},
+		} );
+	};
+
+	return (
+		<Card elevation={ 0 } square={ true } className="all_done">
+			<CardMedia
+				image="https://assets.elementor.com/checklist/v1/images/checklist-step-7.jpg"
```

Comment:
@louiswol94 might know if we need to sanitize this, and how to

Reply:
It's ok, it's hard coded :)

Reply:
perfect